drivers/wifi/infineon/Kconfig.airoc

```
@@ -254,6 +254,11 @@ config CYW955573M2IPA1_SM
 	bool "CYW955573M2IPA1_SM"
 	help
 	  Infineon CYW955573M2IPA1 (SM) module
+
+config CYW955513UBG_SM
```

```suggestion
config CYW55513IUBG_SM
```
Was this the intention?